.github/ISSUE_TEMPLATE/05_onboard.yml

```diff
@@ -0,0 +1,49 @@
+name: Onboarding Tasks
+description: Bringing a Core or non-Core member onto the admiral team
```

Comment:
I assume this issue should be created by Core members only.

Maybe we should mention this.

Reply:
I put message in it